docs/UserGuide.md

```diff
@@ -1,21 +1,21 @@
+![AthletiCLI Banner with fitness imagery](images/AthletiCLI-Banner.png)
 ---
 layout: page
-title: User Guide
+title: AthletiCLi User Guide
```

Comment:
```suggestion
title: User Guide
```

We need to maintain consistency in capitalization, i.e., `AthletiCLI` instead of `AthletiCLi`.

Also, I don't think "AthletiCLI" needs to be included in the title. ([AB3 Example](https://se-education.org/addressbook-level3/UserGuide.html))